plugins/typescript/src/index.ts

```diff
@@ -108,8 +116,20 @@ const plugin: Plugin = {
     );
 
     const queries = Object.entries(schema.queries).map(
-      ([queryName, { operationId }]) => {
-        const functionName = camelCase(operationId);
+      ([queryName, schemaValue]) => {
+        const functionName = camelCase(schemaValue.operationId);
+
+        if (schemaValue.extra) {
+          const extra = schemaValue.extra;
+          const extraString = dedent`${JSON.stringify(extra)}`;
+
+          return dedent`
+          ${functionName}(req) {
+              const extra = ${extraString}
+              return driver.onQueried("${queryName}", req, extra)
```

Comment:
```suggestion
              return driver.onQueried("${queryName}", req, {
                extra,
              })
```
How about leaving the availabiliy of properties more than `extra` for third parameter?